utilities/ovs-vsctl.c

```diff
@@ -57,11 +57,6 @@
 
 VLOG_DEFINE_THIS_MODULE(vsctl);
 
-#ifdef P4OVS
```

Comment:
The _license_ is just fine. It's the Apache 2.0 license, not an Intel license. OvS is licensed under Apache 2.0, and the notice was present in the file before we modified it.

What we might need to remove is the Intel _copyright notice_.

The copyright notice and the license are two different things, governed by two different sets of laws.

If this change set removes _some_ of the Intel modifications from the file, then the Intel copyright notice should remain. If it removes _all_ the Intel modifications, then we should also remove the Intel copyright notice, since it no longer contains any of our intellectual property.

Reply:
Oh my bad, I meant intel copyright not License.

Yes, looks like this PR is removing all Intel added code.

Reply:
Removed Intel copyright